modules/permutiveRtdProvider.md

```diff
@@ -31,39 +32,117 @@ pbjs.setConfig({
+
+## Cohort Activation with Permutive RTD Module
+
+### _Enabling Standard Cohorts_
+
+**Note**: Publishers must be enabled on the above Permutive RTD Submodule to enable Standard Cohorts.
+
+The acbidders config in the Permutive RTD module allows publishers to determine which demand partners (SSPs) will receive standard cohorts via the <u>user.data</u> ortb2 object. Cohorts will be sent in the `p_standard` key-value.
+
+The Permutive RTD module sets standard cohort IDs as bidder-specific ortb2.user.data first-party data, following the Prebid ortb2 convention.
+
+There are **two** ways to assign which demand partner bidders (e.g. SSPs) will receive Standard Cohort information via the Audience Connector (acbidders) config:
+
+#### Option 1 - Automated
+
+New demand partner bidders may be added to the acbidders config directly within the Permutive Platform.
+
+**Permutive can do this on your behalf**. Simply contact your Permutive CSM with strategicpartnershipops@permutive.com on cc,
+indicating which bidders you would like added.
+
+Or, a publisher may do this themselves within the UI using the below instructions.
+
+##### Create Integration
+
+In order to update acbidders via the Permutive dashboard,
+it is necessary to first enable the prebid integration in the integrations page (settings).
+
+**Note on Revenue Insights:** The prebid integration includes a feature for revenue insights,
+which is not required for the purpose of updating acbidders config.
+Please see [this document](https://support.permutive.com/hc/en-us/articles/360019044079-Revenue-Insights) for more information about revenue insights.
+
+##### Update acbidders
+
+The input for the “Data Provider config” is currently a multi-input free text.
+A valid “bidder code” needs to be entered in order to enable Standard Cohorts to be passed to the desired partner.
+The [prebid Bidders page](https://docs.prebid.org/dev-docs/bidders.html) contains instructions and a link to a list of possible bidder codes.
+
+Acbidders can be added or removed from the list using this feature, however, this will not impact any acbidders that have been applied using the manual method below.
+
+#### Option 2 - Manual
+
+As a secondary option, new demand partner bidders may be added manually.
+
+To do so, a Publisher may define which bidders should receive Standard Cohorts by
+Including the _ID_ of any bidder in the `acBidders` array.
+
+**Note:** If a Publisher ever needs to remove a manually-added bidder, the bidder will also need to be removed manually.
+
+### _Enabling Custom Cohort IDs for Targeting_
+
+Separately from Standard Cohorts - The Permutive RTD module also supports passing any of the **custom** cohorts created in the dashboard to some SSP partners for targeting
+e.g. setting up publisher deals. For these activations, cohort IDs are set in bidder-specific locations per ad unit (custom parameters).
+
+Currently, the supported bidders for custom cohort targeting are:
```

Comment:
Perhaps "bidders with known support for custom cohort targeting" ?

Reply:
Due to the fact that we have to enable bidders for custom targeting on our side, I think we have control on this?

Reply:
It is a rules violation to favor a bidder; all bidders should be able to see your cohorts if a publisher chooses to send them there explicitly. 

One reason for this is suppose you wanted to add support for say, Pubmatic, a publisher should not have to upgrade their Prebid software to achieve this. It is fair to say only known bidders understand the cohorts.

Reply:
Also, you should be aware we plan to remove mentions of appnexus and magnite from your module in the near future, https://github.com/prebid/Prebid.js/issues/8596

Reply:
To clarify, remove from your adapter but add to the other adapters, eg if magnite expects your segments on params.permutive, it needs to have that code in the magnite adapter.

Reply:
I see, thanks for the info - it makes sense. I am going to share this with our product team to discuss / prioritize 👍